src/unicodeRegexes.js

```diff
@@ -8,10 +9,5 @@ const hangulRegex = /[\uAC00-\uD7AF]/;
 // - Hangul syllables: [\uAC00-\uD7AF]
 // - Fullwidth punctuation: [\uFF00-\uFF60]
 // Notably missing are halfwidth Katakana and Romanji glyphs.
-const cjkRegex =
+export const cjkRegex: RegExp =
```

Comment:
This is redundant; you don't have to explicitly annotate consts for Flow to recognize their type.

Reply:
Removed. And updated commit description.